dev/top_printers.ml

```diff
@@ -515,7 +515,7 @@ open Egramml
 
 let _ =
   try
-    Vernacinterp.vinterp_add false ("PrintConstr", 0)
+    Vernacextend.vinterp_add false ("PrintConstr", 0)
```

Comment:
TBH I don't know what this stuff is for, maybe we could just remove it and nobody would notice.

Reply:
I guess it is used to Print a Constr from Coq? I'd guess @herbelin uses it.

Reply:
I would support that. I don't use them. @herbelin probably does though.

Reply:
(Nice race condition.)